qutebrowser/keyinput/basekeyparser.py

```diff
@@ -315,6 +318,13 @@ def _handle_result(self, info: keyutils.KeyInfo, result: MatchResult) -> None:
             assert result.command is not None
             self._debug_log("Definitive match for '{}'.".format(
                 result.sequence))
+
+            max_digits = sys.get_int_max_str_digits()
+            if len(self._count) > max_digits:
+                message.error(f"Count limit exceeded ({max_digits} digits)",
+                              stack=traceback.format_exc())
```

Comment:
This only makes sense inside an exception handler, it will just return `NoneType: None` outside of an `except ...:`.

Since we aren't handling an exception here, just remove it instead.

Reply:
Oh yes, of course. Thank you.

Initially it was EAFP:
```python
try:
    count = int(self._count) if self._count else None
except ValueError:
    message.error(f"Count limit exceeded ({max_digits} digits)", stack=traceback.format_exc())
    return
```
Then I felt a LBYL check would be more explicit, but completely missed that.